app/services/send_welcome_email_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SendWelcomeEmailService
+  def self.call(current_user:)
+    new.call(current_user: current_user)
+  end
+
+  def call(current_user:)
+    return if current_user.welcome_email_sent_at
+
+    WelcomeEmailMailer.send_welcome_email(
+      first_name: current_user.first_name,
```

Comment:
first_name can potentially be nil 

https://github.com/DFE-Digital/register-trainee-teachers/blob/master/app/services/dfe_sign_in_users/update.rb#L20-L23

cc @stellenberger

Reply:
The `User` model has presence validation on `first_name`
We shouldn't have a user without `first_name` set.